index.bs

```diff
@@ -6368,7 +6390,11 @@ for="value-with-size">value</dfn> and <dfn for="value-with-size">size</dfn>.
  1. Assert: |container| has \[[queue]] and \[[queueTotalSize]] internal slots.
  1. If ! [$IsNonNegativeNumber$](|size|) is false, throw a {{RangeError}} exception.
  1. If |size| is +∞, throw a {{RangeError}} exception.
- 1. [=list/Append=] a new [=value-with-size=] with [=value-with-size/value=] |value| and
+ 1. Let |enqueuedValue| be |value|.
+ 1. If |container|.\[[isTransferring]] is true, perform the following steps:
```

Comment:
`EnqueueValueWithSize` is also used by `WritableStreamDefaultControllerWrite` and `WritableStreamDefaultControllerClose`, but `WritableStreamDefaultController` does not have an `[[isTransferring]]` slot.

I think we should move the transferring logic out of `EnqueueValueWithSize`, and instead put it at the call site in `ReadableStreamDefaultControllerEnqueue`.

Reply:
If no slot, then my understanding is that the value is undefined, which is different from true.
That said, the end plan is to use transfer for writable streams as well, which would probably allow reusing the same 1EnqueueValueWithSize`.

Reply:
I updated to `if |container| has a \[[isTransferring]] slot whose value is true` to make that clearer